go.mod

```diff
@@ -45,16 +45,15 @@ require (
 	github.com/mrunalp/fileutils v0.5.0
 	github.com/nxadm/tail v1.4.8
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/gomega v1.19.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198
 	github.com/opencontainers/runc v1.1.0
 	github.com/opencontainers/runtime-spec v1.0.3-0.20211214071223-8958f93039ab
 	github.com/opencontainers/runtime-tools v0.9.1-0.20220110225228-7e2d60f1e41f
-	github.com/opencontainers/selinux v1.10.0
+	github.com/opencontainers/selinux v1.10.1
```

Comment:
You have to remove this change to fix the tests, see https://github.com/containers/podman/pull/13687 https://github.com/containers/buildah/pull/3875

Reply:
@Luap99 I think we might wanna revert version bump of selinux in `c/common` cause `make vendor` will automatically bump `selinux`. Could you review my above PR.